neuraxle/base.py

```diff
@@ -1933,20 +1860,21 @@ def apply(self, method_name: str, step_name=None, *kargs, **kwargs) -> Dict:
 
         return results
 
-    def apply_method(self, method: Callable, step_name=None, *kargs, **kwargs) -> Union[Dict, Iterable]:
+    def apply_method(self, method: Callable, step_name=None, children_only=False, *kargs, **kwargs) -> Union[Dict, Iterable]:
         """
         Apply method to the meta step and its wrapped step.
 
         :param method: method to call with self
         :param step_name: step name to apply the method to
+        :param children_only: apply method to children only
         :param kargs: any additional arguments to be passed to the method
         :param kwargs: any additional positional arguments to be passed to the method
         :return: accumulated results
         """
-        results = BaseStep.apply_method(self, method=method, step_name=step_name, *kargs, **kwargs)
+        results = BaseStep.apply_method(self, method=method, step_name=step_name, children_only=children_only, *kargs, **kwargs)
 
         if step_name is not None:
-            step_name = "{}__{}".format(step_name, self.name)
+            step_name = "{}{}{}".format(step_name, RECURSIVE_STEP_SEPARATOR, self.name)
```

Comment:
children_only seems to be ignored here. Is this argument properly handled? it should be revised I think, yet I don't have a solution